backend/app/views/spree/admin/taxons/_form.html.erb

```diff
@@ -4,12 +4,12 @@
       <%= f.field_container :name, class: ['form-group'] do %>
         <%= f.label :name, Spree.t(:name) %> <span class="required">*</span>
         <%= text_field :taxon, :name, class: 'form-control' %>
-        <%= f.error_message_on :name, class: 'error-message' %>
+        <%= f.error_message_on :name, class: 'error-message', value: @taxon.name %>
```

Comment:
@charliestarkpl There is no need to explicitly set ```:value``` attr, the ```FormBuilder``` object will handle that for you. https://apidock.com/rails/ActionView/Helpers/FormHelper/form_for

> For fields generated in this way using the FormBuilder, if :person also happens to be the name of an instance variable @person, the default value of the field shown when the form is initially displayed (e.g. in the situation where you are editing an existing record) will be the value of the corresponding attribute of @person.

It concerns all the changes introduced in this file. In my opinion you could drop this commit :-)